.gitignore

```diff
@@ -0,0 +1,12 @@
+# python stuff
+*.py[cod]
```

Comment:
what does the `[cod]` part mean ? is this like a regexp expanding to `*.pyc / *.pyo / *.pyd` ?

Reply:
yes; it's inspired from [this generic gitignore config](https://github.com/github/gitignore/blob/master/Python.gitignore).